index.html

```diff
@@ -989,6 +1093,9 @@ <h3>Role Mapping Table</h3>
 				</tr>
 				<tr id="role-map-img">
 					<th><a class="role-reference" href="#img"><code>img</code></a></th>
+					<td class="role-computed">
+						<p><code>image</code></p>
+					</td>
```

Comment:
Patch also needs an whole TR entry for `image`... I guess we hadn't noticed it was missing... (`none` and `presentation` are already included)

Ideally the `img` row would just point to the entire `image` row for all columns, to avoid duplication. Ditto for the `presentation` row pointing to the `none` row.

Reply:
That is true. Unfortunately the table, and it's automatic conversion to the summary/details, doesn't make that easy... but to be honest, I can't say I ever use the "single table view" and it's a bit of a pain. I think I'll add an issue to discuss removing it.

Reply:
I'm going to fix the missing entry for image after I remove the table view, because I'm motivated to remove the table view right now: https://github.com/w3c/core-aam/issues/169